.github/actions/setup-rocm/action.yml

```diff
@@ -87,4 +87,4 @@ runs:
       shell: bash
       run: |
         # All GPUs are visible to the runner; visibility, if needed, will be set by run_test.py.
-        echo "GPU_FLAG=--device=/dev/mem --device=/dev/kfd --device=/dev/dri --group-add video --group-add daemon" >> "${GITHUB_ENV}"
+        echo "GPU_FLAG=--device=/dev/mem --device=/dev/kfd --device=/dev/dri --group-add video --group-add daemon --group-add bin" >> "${GITHUB_ENV}"
```

Comment:
The group-add daemon and group-add bin are needed in the Ubuntu 24.04 and Almalinux OSs. This is due to the device files (/dev/kfd & /dev/dri) being set to video for its group. This video group subgid is set to the subgid value in image. This can change depending on the OS so both are necessary.